src/vg_client_pool.erl

```diff
@@ -73,20 +74,17 @@ get_pool(Topic, RW, Try) ->
     case ets:lookup(?topic_map, Topic) of
         [] ->
             case {RW, Try} of
-                {write, _} ->
-                    %% just make it up for now  TODO: make this a settable
-                    %% behavior serverside
-                    [{_, Default}] = ets:lookup(?topic_map, <<>>),
-                    lager:debug("default pool: ~p ~p", [Topic, Default]),
-                    {ok, make_pool_name(Default, RW)};
+                {write, first} ->
+                    vg_client:ensure_topic(Topic),
+                    refresh_topic_map(),
+                    get_pool(Topic, RW, second);
                 {read, first} ->
                     refresh_topic_map(),
                     get_pool(Topic, RW, second);
-                {read, second} ->
+                {_, second} ->
                     lager:debug("not found chain: ~p", [Topic]),
```

Comment:
I messed up here, this should be a logged error, I think, with a sensible message rather than my weird comment which made sense at the time.